porch/func/Dockerfile

```diff
@@ -17,11 +17,11 @@ FROM gcr.io/kpt-fn/apply-setters:v0.1.1 as apply-setters-v1
 FROM gcr.io/kpt-fn/ensure-name-substring:v0.1.1 as ensure-name-substring
 FROM gcr.io/kpt-fn/search-replace:v0.2.0 as search-replace
 FROM gcr.io/kpt-fn/set-annotations:v0.1.4 as set-annotations
-FROM gcr.io/kpt-fn/set-image:v0.1.0 as set-image
+FROM gcr.io/kpt-fn/set-image:v0.1.1 as set-image
 FROM gcr.io/kpt-fn/set-labels:v0.1.5 as set-labels
-FROM gcr.io/kpt-fn/set-namespace:v0.2.0 as set-namespace
+FROM gcr.io/kpt-fn/set-namespace:v0.4.1 as set-namespace
 FROM gcr.io/kpt-fn/set-project-id:v0.2.0 as set-project-id
-FROM gcr.io/kpt-fn/starlark:v0.3.0 as starlark
```

Comment:
@mengqiy Do When I tried bumping the starlark version to the latest v0.4.3, [ln56](https://github.com/GoogleContainerTools/kpt/pull/3088/files#diff-b05c9366bfa369d0e8ba62afabd334883c3edf7a146c11aa25bd5cfd9bd8ff83R56) shows 
```
error: failed to solve: failed to compute cache key: "/usr/local/bin/star" not found: not found
```

Is this starlark v0.3.0 a special version which is tied to the proch/func/Dockerfile?